app/src/main/java/io/intrepid/contest/splash/SplashContract.java

```diff
@@ -0,0 +1,18 @@
+package io.intrepid.contest.splash;
+
+import io.intrepid.contest.base.BaseContract;
+
+public interface SplashContract {
+
+    interface View extends BaseContract.View {
+        void showCreateContestScreen();
+
```

Comment:
I mentioned this with Gabi too, I prefer to remove the spaces but its up to you two what you want to do.